lib/opal/cli_runners.rb

```diff
@@ -77,5 +77,26 @@ def self.register_legacy_runner(klass_name, *names)
     register_legacy_runner :Nashorn,     :nashorn
     register_legacy_runner :Nodejs,      :nodejs, :node
     register_legacy_runner :Server,      :server
+
+    # @elia Is this what you had in mind ?
```

Comment:
@elia ping :thinking:

Reply:
Yeah, keep in mind that it can be done with modules/classes too:

```rb
# cli_runners/nodejs.rb

  module Opal
    module CliRunners
      class Nodejs
        def self.call(data)
          # …
        end
      end
    end
  end
```

```rb
# cli_runners.rb
register_runner :nodejs, Nodejs
```

Reply:
> Yeah, keep in mind that it can be done with modules/classes too:

I will do that